_health-care/_js/_components/social-security-number.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import _ from 'lodash';
+
+import TextInputRequired from './text-input-required';
+
+/**
+ * A component for social security number. Includes component for required text input. 
+ * Adds error markup if in an error state: error class, error id for aria, and 
+ * ErrorMessage component. 
+ *
+ * @constructor
+ */
+class SocialSecurityNumber extends React.Component {
+  constructor() {
+    super();
+    // In general, I think we need to store the error state at the top level component, 
+    // like we are with form data, and create the one-way data flow with that as well. 
+    // This may solve the problems we're having with fields not registering errors 
+    // on page refresh.
+    this.state = {hasError: false};
+    this.handleErrorChange = this.handleErrorChange.bind(this);
+  }
+
+  // TODO: What this really needs to do is loop through the inputs and if any have
+  // a state of true it needs to set the error state to true
+  // Also figure out why this isn't actually getting called the first time around
```

Comment:
You will need to call it explicitly after props have been set...possibly in componentWillMount(), but let's file a bug and reference the number in a TODO here.  It requires a bit more understanding of the component lifecycle than I think we should get into.

See https://facebook.github.io/react/docs/component-specs.html